{
  "dismiss_case": {
    "success": "The case has been dismissed successfully.",
    "error": "Error occurred while dismissing the case."
  },
  "flag_case": {
    "success": "The case has been flagged successfully.",
    "error": "Error occurred while flagging the case."
  },
  "approve_case": {
    "success": "The case has been approved successfully.",
    "error": "Error occurred while approving the case."
  },
  "reject_case": {
    "success": "The case has been rejected successfully.",
    "error": "Error occurred while rejecting the case."
  },
  "ask_revision_case": {
    "success": "Email sent successfully.",
    "error": "Error occurred while sending email."
  },
  "approve_document": {
    "success": "The document has been approved successfully.",
    "error": "Error occurred while approving the document.\n\n{{errorMessage}}"
  },
  "reject_document": {
    "success": "The document has been rejected successfully.",
    "error": "Error occurred while rejecting the document.\n\n{{errorMessage}}"
  },
  "ask_revision_document": {
    "success": "The document has been asked for revision successfully.",
    "error": "Error occurred while asking for revision of the document.\n\n{{errorMessage}}"
  },
  "revert_revision": {
    "success": "Asking for document revision has been cancelled successfully.",
    "error": "Error occurred while reverting the document revision.\n\n{{errorMessage}}"
  },
  "update_document_properties": {
    "success": "The document has been updated successfully.",
    "error": "Error occurred while updating the document.\n\n{{errorMessage}}"
  },
  "assign_case": {
    "success": "The case has been assigned to {{assignee}} successfully.",
    "error": "Error occurred while assigning the case to {{assignee}}."
  },
  "unassign_case": {
    "success": "The case has been unassigned successfully.",
    "error": "Error occurred while unassigning the case."
  },
  "copy_to_clipboard": "Copied to clipboard: {{text}}",
  "ocr_document_error": "Error occurred while performing OCR on the document.",
  "validation_error": "‚ùå Validation error",
  "event": {
    "success": "Event sent successfully.",
    "error": "Error occurred while sending event."
  },
  "case_creation": {
    "success": "Case created successfully.",
    "error": "Error occurred while creating a case."
  }
}
